src/test/resources/MethodInEnum/expected/com/example/Foo.java

```diff
@@ -18,9 +18,7 @@ public static void testing() {
         }
         ;
 
-        public abstract void testing() {
-            throw new Error();
-        }
+        public abstract void testing();
```

Comment:
This one was originally a method signature. The old version of Specimin added a `throw new Error()` body by mistake.